include/deal.II/base/point.h

```diff
@@ -201,96 +202,13 @@ class Point : public Tensor<1, dim, Number>
    * convertible to @p Number.
    */
   template <typename OtherNumber>
-  Point<dim, Number> &
+  constexpr Point<dim, Number> &
   operator=(const Tensor<1, dim, OtherNumber> &p);
 
   /**
-   * @name Addition and subtraction of points.
+   * @name Norms and distance between points
    * @{
    */
-
-  /**
-   * Add an offset given as Tensor<1,dim,Number> to a point.
-   *
-   * @note This function can also be used in @ref GlossDevice "device" code.
-   */
-  DEAL_II_HOST_DEVICE Point<dim, Number>
-                      operator+(const Tensor<1, dim, Number> &) const;
```

Comment:
I don't think you can just remove this function. If you do, the compiler will choose the `operator+` in the base class, which returns a `Tensor<1,dim>`, not a `Point<dim>`. Or do I miss something?